packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -1082,6 +1082,17 @@ export class UserInputUtil {
         }
     }
 
+    public static async failedNetworkStart(popupMsg: string, logMsg: string, outputAdapter: VSCodeBlockchainOutputAdapter): Promise<void> {
+
+         const moreInfo: string = 'More Details';
```

Comment:
Might be worth making this a static variable at the top of `UserInputUtil.ts` as I expect we might use this multiple times in future.

E.g
`static readonly MORE_DETAILS: string = 'More Details'`

This means that in our tests we can use this variable as well - so if the text ever changes, the tests will be valid as well.